sample/src/main/AndroidManifest.xml

```diff
@@ -28,22 +28,25 @@
     <activity android:name=".DialogActivity"/>
     <activity android:name=".PreferencesActivity"/>
     <activity android:name=".DatabaseActivity"/>
+    <activity android:name=".FileActivity">
+      <intent-filter>
+        <action android:name="android.intent.action.MAIN"/>
```

Comment:
At Barista, the main Activity is just a way to ease developers the task to develop an Activity that helps testing it. Please, don't focus the review on this change.